components-java-sdk/src/test/java/io/dapr/components/domain/state/options/StateConsistencyTest.java

```diff
@@ -0,0 +1,53 @@
+package io.dapr.components.domain.state.options;
+
+import dapr.proto.components.v1.State;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class StateConsistencyTest {
+
+  @Test
+  void sanityCheckWithKnownEquivalent() {
+    final StateConsistency strongConsistency = StateConsistency.STRONG;
+    assertThat(strongConsistency.toProto())
+        .isNotNull()
+        .isEqualTo(State.StateOptions.StateConsistency.CONSISTENCY_STRONG);
+  }
+
+  @ParameterizedTest
+  @EnumSource
```

Comment:
Ooh, I wish Xunit had that!

Reply:
Pretty neat, eh? Given how similar the tests are I wish I could generalize this even further but I guess it is fine as it is right now.